.github/workflows/release.yml

```diff
@@ -0,0 +1,106 @@
+name: Release
+on:
+  push:
+    branches:
+      - master
+      - series/*
+    tags: 
+      - '*'
+env:
+  HUGO_VERSION: 0.26
+  SCALA_VERSION: 2.12.10
```

Comment:
I wish we weren't duplicating these with ci.yml.  Ideas?  We could use secrets, but that would be an abuse.

Reply:
This duplication has already bothered me in http4s-jdk-http-client. Concerning this (and more generally, all YAML shortcomings): One could generate these config files via [Dhall](https://dhall-lang.org/) (there are already approaches for this in several languages, see e.g. [here](https://github.com/vmchale/github-actions-dhall)). This config could even be shared between multiple (http4s) repos (integrity is ensured).

Downsides:
 - Contributing to the CI scripts requires Dhall knowledge (on the other hand, translating YAML changes to the Dhall source files should be very easy, and it *should* be relatively rare).

Advantages:
 - No YAML surprises (we already had that in http4s-jdk-http-client...). I think it is safe to claim that Dhall is superior from a design perspective.
 - DRY: Sharing of variables/snippets between files or even between repos

I could try to create an example for http4s-jdk-http-client to see if this is an approach worth pursuing.

Reply:
Ha! I mentioned Dhall on the previous ticket (#3147).  It's our strategy at work for escaping YAML Hell.

One other downside is that we'd be forced to keep generated code in our repositories.  I strongly dislike having a source of truth and a second git-managed reflection of that truth.  But maybe it's no worse than copying and pasting, and a having a Dhall utility to generate these could pay off as the integrations spin off into separate repos and as we grow more. Maybe we should see if other maintainers are enthused or horrified by the idea.

The other way we're winning at work is putting as much logic as possible into ci.nix, and the YAML remains thin and rarely changes.  The equivalent in our world would be to do what we can in sbt, or even standardly named shell scripts.  But part of the reason ours are so big here is because we're splitting into multiple jobs for greater concurrency.